sources/web/datalab/static/util.js

```diff
@@ -1,7 +1,14 @@
 define(() => {
+
+  window.datalab = window.datalab || {};
+  window.datalab.debug = false;
```

Comment:
That doesn't seem sufficient if you need to debug something that happens when the page first loads.

I would suggest something like a 'data-debug-enabled' attribute to the body of all of our templates and then set that based on an environment variable when the template gets turned into HTML